build-release-tools/application/make_debian_packages.py

```diff
@@ -230,20 +247,45 @@ def write_downstream_parameter_file(build_directory, manifest_file, is_official_
             params['RACKHD_COMMIT'] = rackhd_commit
         else:
             raise RuntimeError("commit-id of RackHD is None. Please check the manifest {0}".format(manifest_file))
-            
+        
         # Write downstream parameters to downstream parameter file.
         common.write_parameters(parameter_file, params)
     except Exception, e:
         raise RuntimeError("Failed to write downstream parameter file \ndue to {0}".format(e))
 
+def make_packages_filter(bintray, build_directory, is_official_release):
```

Comment:
from the function name, it seems it has 'make' action in this function, actually it doesn't